extensions/api/participant-context-mgmt-api/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ dependencies {
     api(project(":spi:identity-hub-spi"))
     api(project(":spi:identity-hub-did-spi"))
     implementation(project(":extensions:api:identityhub-management-api-configuration"))
+    implementation(project(":extensions:api:keypair-mgmt-api"))
```

Comment:
@paullatzelsperger  not a fan of what I've done here... But I am also not fan of creating a module just for the `KeyDescriptorValidator` (which is used both by the `participant-context-mgmt-api` and `keypair-mgmt-api` modules)

Reply:
I agree that it is not super elegant... maybe we could introduce a `common-validators` module, where _all_ validators are stored?